### DISCUSSION TITLE
在汉字识别模型上使用数据进行微调，生成的模型在双字上识别率很高，为什么在单字上准确率很低

### DISCUSSION THREAD

**Discussion by 1259666087** [PaddlePaddle/PaddleOCR#12825] [Q&A]
使用了八十万条数据进行训练，每条数据为1-4个汉字组成的图片

**Comment by GreatV** [comment]
试试改一下 `max_text_length` 看看效果会不会变化

**Reply by 1259666087** [reply]
要是想要只识别单字和双字的话把max_text_length设为多少比较好呢，是不是训练集只用双字单字图片比较好

**Reply by GreatV** [reply]
1. 如果只希望识别单字和双字组合，将`max_text_length`设为2是一个合理的选择。这意味着模型最多只会考虑2个字符的文本长度。
2. 在训练集里只使用单字和双字的图片样本，这样可以确保模型专注于识别这些目标。
3. 确保单字和双字样本的比例尽量均衡，这样模型不会偏向于识别某一种类型的样本。
4. 即使只使用单字和双字样本，也要确保样本的多样性，包括不同字体、不同大小、不同噪声水平等，以提高模型的泛化能力。

**Reply by 1259666087** [reply]
请问一下改*max_text_length是在哪里改呢，在Global:
  max_text_length: &max_text_length 25这里改成
max_text_length: &max_text_length 2么
